packaging/os/dnf.py

```diff
@@ -22,14 +23,11 @@
 
 import traceback
 import os
+import operator
+import functools
 import dnf
-
-try:
-    from dnf import find_unfinished_transactions, find_ts_remaining
-    from rpmUtils.miscutils import splitFilename
-    transaction_helpers = True
-except:
-    transaction_helpers = False
+import dnf.cli
+import dnf.util
```

Comment:
this should really use the exception handling you removed, we don't want modules to end in an exception, it should be captured as it was before.